### ISSUE TITLE
LOOT?

### ISSUE THREAD

**Issue by poperigby** [DanaePlays/mlox-rules#154]
Morrowind has a [LOOT masterlist](https://github.com/loot/morrowind). Do you think it would be a good idea to integrate these rules into that? Then we can have a centralized place where Bethesda plugin sorting rules go, and LOOT has a better master list structure in my opinion. What do you think?

**Comment by lucipharis** [comment]
I think this issue would fit better on a LOOT repo. There were some people interested in doing that over there a while ago but they've since abandoned it.

**Comment by poperigby** [comment]
Good idea. I just thought we could coordinate so there isn't duplicate work.
